client/me/purchases/manage-purchase/plan-details/billing-period.jsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External Dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal Dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSelect from 'components/forms/form-select';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import { isMonthly, getYearlyPlanByMonthly } from 'lib/plans/constants';
+import { planItem } from 'lib/cart-values/cart-items';
+import { addItem } from 'lib/upgrades/actions';
+
+class PlanBillingPeriod extends Component {
+	static propTypes = {
+		purchase: PropTypes.object,
+	};
+
+	handleBillingPeriodChange = ( event ) => {
+		if ( event.target.value === 'monthly' ) {
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'Monthly' )` *ES Score: 17*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-plan-purchases-billing-period-switch)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).